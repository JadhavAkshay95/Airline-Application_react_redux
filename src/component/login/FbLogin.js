import React from 'react';
import FacebookLogin from 'react-facebook-login';
import PropTypes from 'prop-types';
import './login.scss';

export const FbLogin = ({ handleLogin }) => {
  return (
    <FacebookLogin
      className="fb-login"
      appId="888455274942887"
      autoLoad={false}
      fields="name,email,picture"
      callback={handleLogin}
      icon="fa-facebook"
    />
  );
};

export default FbLogin;
FbLogin.propTypes = { handleLogin: PropTypes.func };
