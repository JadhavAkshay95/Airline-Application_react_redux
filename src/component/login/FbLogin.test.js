import React from 'react'
import { shallow } from 'enzyme'
import FbLogin from './FbLogin'

const renderFbLogin = args => {
  const defaultProps = { handleLogin: () => {} }
  const props = { ...defaultProps, ...args }
  return shallow(<FbLogin {...props} />)
}

describe('Render Facebook Login component', () => {
  it('Render flight list', () => {
    const wrapper = renderFbLogin()
    expect(wrapper).toMatchSnapshot()
  })
})
