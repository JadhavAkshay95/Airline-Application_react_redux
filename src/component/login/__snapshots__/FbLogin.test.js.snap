// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Facebook Login component Render flight list 1`] = `ShallowWrapper {}`;
